### PR TITLE
Release plugin 3 13 2

### PR DESCRIPTION
**Pull request by kate0**
GE Plugin after release activities

### Context
Update the released plugin version in the following file:
- [org.gradle.smoketests.BuildScanPluginSmokeTest.groovy](https://github.com/gradle/gradle/blob/release/subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy)
- [org.gradle....autoapply.AutoAppliedGradleEnterprisePlugin.java](https://github.com/gradle/gradle/blob/release/subprojects/core/src/main/java/org/gradle/plugin/management/internal/autoapply/AutoAppliedGradleEnterprisePlugin.java)
- [settings.gradle.kts](https://github.com/gradle/gradle/blob/release/settings.gradle.kts)
- [build-logic-commons/gradle-plugin/build.gradle.kts](https://github.com/gradle/gradle/blob/release/build-logic-commons/gradle-plugin/build.gradle.kts)
- [build-logic/build-platform/build.gradle.kts](https://github.com/gradle/gradle/blob/release/build-logic/build-platform/build.gradle.kts)
- [github-actions.adoc](https://github.com/gradle/gradle/blob/master/subprojects/docs/src/docs/userguide/reference/ci-systems/github-actions.adoc?plain=1#L68)

### CODE REVIEW

File: subprojects/docs/src/docs/userguide/running-builds/build_environment.adoc [hunk 1]
```diff
@@ -53,6 +53,21 @@ _By default, the build cache is *not* enabled._
 When set to true, individual input property hashes and the build cache key for each task are logged on the console. Learn more about <<build_cache.adoc#sec:task_output_caching, task output caching>>.
 _Default is `false`._
 
+`org.gradle.configuration-cache=(true,false)`::
```
Comment (guylabs) [inline subprojects/docs/src/docs/userguide/running-builds/build_environment.adoc:56]: Is that change needed?

**Review by guylabs — APPROVED** [review]

**Review by aeiou-st — APPROVED** [review]
PR

**Comment by kate0** [comment]
@bot-gradle test and merge

File: subprojects/docs/src/docs/userguide/running-builds/build_environment.adoc (hunk 1 above)
Comment (kate0) [reply subprojects/docs/src/docs/userguide/running-builds/build_environment.adoc:56]: @abstratt I have your commit tagged along in my PR, may you please check it and confirm we are good to merge it altogether?

**Review by aeiou-st — APPROVED** [review]

File: subprojects/docs/src/docs/userguide/running-builds/build_environment.adoc (hunk 1 above)
Comment (abstratt) [reply subprojects/docs/src/docs/userguide/running-builds/build_environment.adoc:56]: @kate0 Sorry, only saw this now. I must admit I don't understand how my commit ended up in your branch.
